straxen/plugins/peaklet_processing.py

```diff
@@ -26,8 +26,8 @@
                  # for more information
                  default=(
                      None,  # Reserved
-                     ((0.5, 1), (4, 0.4)),
-                     ((2, 1), (4.5, 0.4))),
+                     ((0.5, 1.), (6.0, 0.4)),
```

Comment:
This is a typo, right?

Reply:
Sorry, I missed that plot.

Reply:
@zhut19 do you think this PR can be merged now?